utils/resolve.js

```diff
@@ -34,7 +34,11 @@ function tryRequire(target, sourceFile) {
   try {
     // Check if the target exists
     if (sourceFile != null) {
-      resolved = createRequire(path.resolve(sourceFile)).resolve(target)
+      try {
+        resolved = createRequire(path.resolve(sourceFile)).resolve(target)
+      } catch {
```

Comment:
bindingless catch isn't covered by our babel settings atm
```suggestion
      } catch (e) {
```